config_defaults_inc.php

```diff
@@ -4615,6 +4621,7 @@
 	'time_tracking_view_threshold',
 	'time_tracking_with_billing',
 	'time_tracking_without_note',
+	'time_tracking_cost_per_hour',
```

Comment:
Please insert before `time_tracking_edit_threshold` to keep the list alphabetically sorted.